### DISCUSSION TITLE
FOSS 课程讨论汇总贴

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#156] [3️⃣ 聊学术]
可以在此“Discussion”下讨论与FOSS课程相关的问题，具体日程可以[参见这里](https://github.com/X-lab2017/open-wonderland/blob/main/%F0%9F%8C%B32022-XSOSI-OpenCourse.md)~

**Comment by will-ww** [comment]
8 月 1 日 的第一节课“Overview and History”，欢迎 @xiaoya-yaya 分享，欢迎大家在此条目下进行讨论~

**Comment by xgdyp** [comment]
补充材料：
1. 关于洋葱模型，https://github.com/X-lab2017/open-wonderland/discussions/150  这个discussion中讨论了 non-tech相关的角色
2. https://dirkriehle.com/ 里面有一些补充的论文资料

**Comment by xgdyp** [comment]
Developing Tool -Software forge补充：
1. https://github.com/orgs/X-lab2017/discussions/146 介绍了一些平替软件
2. awesome opensource 一个全的github项目分类库
3. 国内的OSChina维护了一个开源软件分类库

**Reply by xgdyp** [reply]
之前泰炜也分享过一个，我忘记名字了， @TieWay59 你还记得吗

**Reply by TieWay59** [reply]
忘辽

**Comment by will-ww** [comment]
把庄老师昨天分享的几个文章也发下面：
- [三代开源社区的协作模式](http://zhuangbiaowei.github.io/thinking/it/2016/01/17/gens-collaborative-model.html)
- [开源协作工具的演进方向](http://zhuangbiaowei.github.io/opensource/tools/2022/07/17/evolution-of-open-source-development-tools.html)
- [从28万个开源项目中，我们能够学到一些什么？](http://zhuangbiaowei.github.io/thinking/it/2015/05/12/Open-Source-Project-Analysis.html)

**Comment by xgdyp** [comment]
1. 501 c3条款实际上是要遵守一些出口限制的
![image](https://user-images.githubusercontent.com/37795442/184385523-80d2dbd8-f7da-4df8-95e8-09d00f4a679e.png)
2. 分享是所讲的论文 https://dirkriehle.com/wp-content/uploads/2009/12/2009-12-11-Dirk-Riehle-Economic-Case-Open-Source-Foundations-for-Web.pdf
3. 社区分支相关discussion #46
